services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -250,6 +230,16 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
                 dynamic_sidecar_endpoint
```

Comment:
possible states:
- nothing
- created
- running

Current use-cases:
- sim4life issue: 2 out 3 services (created stuck), 3rd service is missing -> continuously raises
-> showing "usually not an issue". -> workaround: admin removes dy-sidecar/proxy, user must close tab and re-open study (admin cannot force start because 1 service is missing)
- iseg issue: 2 out of 2 services (created stuck) -> go through without exception, but nothing will happen -> admin removes dy-sidecar/proxy, user must close tab and re-open study (admin cannot force start because mismatch in docker networks)
- jupyterlab issue: 1 out of 1 service (created stuck): admin force start

Reply:
I think we have fixes for sim4life and jupyterlab incoming to master.
to test iSeg one we need to move prep2go and bladder control to master to have better testing scenarios